application/database/database_routes.py

```diff
@@ -281,7 +281,6 @@ def update_location(location_id):
   """
```

Comment:
@iloveicedgreentea i may be missing it, but the code does not update the `updated_on` field when a `PUT` request succeeds